lib/jellyfish_web/server_socket.ex

```diff
@@ -123,14 +126,16 @@ defmodule JellyfishWeb.ServerSocket do
 
     room_state = get_room_state(option)
 
-    msg =
-      {:subscription_response,
-       %SubscriptionResponse{
+    {:ok, %SubscriptionResponse{id: id, content: room_state}}
+  end
+
+  defp handle_subscribe(%SubscribeRequest{
          id: id,
-         content: room_state
-       }}
+         event_type: {:metrics, %SubscribeRequest.Metrics{}}
+       }) do
+    :ok = Phoenix.PubSub.subscribe(Jellyfish.PubSub, "metrics")
 
-    {:ok, msg}
+    {:ok, %SubscriptionResponse{id: id}}
```

Comment:
I see that we aren't passing the `content` field here, how does that work? Assuming that we are able to ignore this field, it seems little ambiguous, but I cannot think of a better alternative (one more field to `oneof` that is empty and means "response for metrics request"?, tough to tell)

Reply:
According to proto docs, oneof means zero or one of the options, so IMO it's appropriate to just not fill the content field of the message.

Reply:
Fair enough